README.md

```diff
@@ -1,14 +1,31 @@
 # Project Bueno <sub>(temporary name)</sub>
 
-## Bueno's goal is to become what Deno originally was meant to be, a good JS/TS runtime.
+Bueno is a web-standards focused JS/TS runtime with as much tooling built-in as
+possible. It is meant to be both a learning project and a sort of spiritual
+sibling to Deno 1.x. Bueno is very ambitious and has many goals, but one of the
+non-goals is support for node modules in any way (outside of transpilation by
+CDNs).
 
-This isn't true anymore, Deno supports many features that make it basically what
-it originally sweared not to be, e.g.: `npm:`, `node:` specifiers and soon to be
-`deno:` specifier, `package.json` support and more.
+## Goals
 
-## Goals in points:
-
-- No node/npm support IN ANY SHAPE OR FORM
 - Native support TypeScript/TSX/JSX
 - Support as many Web Standard API's as possible
 - URL Imports
+- Go fast
+
+## Anti-Goals
+
+- No node/npm support IN ANY SHAPE OR FORM
```

Comment:
This sounds very confusing due to this bit being a double negative with the "anti-goal" and the "no" at the beginning of the 2 anti-goals

Reply:
Good point. I'll fix this and add a "Why not bun?" section.

Reply:
Cuz fuck bun :)